dvid_point_cloud/sampling.py

```diff
@@ -214,6 +215,25 @@ def uniform_sample(server: str, uuid: str, label_id: int,
         return pd.DataFrame(points_xyz, columns=["x", "y", "z"])
     else:
         return points_xyz
+    
+def label_at_point(server: str, uuid: str, instance: str, point: tuple[int, int, int], supervoxels: bool = False) -> int:
+    """
+    Get the label at a specific point.
+    """
+    client = DVIDClient(server)
+    response = client.get_label(uuid, instance, point, supervoxels=supervoxels)  
+
+    # parse json and read "label" field
```

Comment:
I think this should be moved from sampling to the client. Client package should deal with any kind of JSON interpretation and return just the uint64, not the JSON bytes.